blueocean-rest-impl/src/test/java/io/jenkins/blueocean/service/embedded/rest/NodeDownstreamBuildActionTest.java

```diff
@@ -0,0 +1,17 @@
+package io.jenkins.blueocean.service.embedded.rest;
+
+import io.jenkins.blueocean.rest.hal.Link;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class NodeDownstreamBuildActionTest {
```

Comment:
It will be good to have test where a free style job is triggered from a pipeline. You can refer to PipelineNodeTest for existing functional test.

Reply:
Will do